? lib/vc_dll
? src/regex/regex___Win32_Debug_Unicode
? src/regex/regex___Win32_Debug_Unicode_DLL
? src/regex/regex___Win32_Release_Unicode
? src/regex/regex___Win32_Release_Unicode_DLL
? src/regex/regex___Win32_Release_With_Debug_Info
Index: include/wx/dynload.h
===================================================================
RCS file: /pack/cvsroots/wxwidgets/wxWidgets/include/wx/dynload.h,v
retrieving revision 1.19
diff -u -r1.19 dynload.h
--- include/wx/dynload.h	2004/02/05 10:37:56	1.19
+++ include/wx/dynload.h	2004/04/28 19:42:50
@@ -126,7 +126,7 @@
     {
         Load(libname, flags);
     }
-    ~wxPluginManager() { Unload(); }
+    ~wxPluginManager() { if ( IsLoaded() ) Unload(); }
 
     bool   Load(const wxString &libname, int flags = wxDL_DEFAULT);
     void   Unload();

 	  	 
